pom.xml

```diff
@@ -89,7 +89,7 @@
       <dependency>
         <groupId>io.jenkins.tools.bom</groupId>
         <artifactId>bom-2.289.x</artifactId>
-        <version>987.v4ade2e49fe70</version>
+        <version>1135.va_4eeca_ea_21c1</version>
```

Comment:
Do you need the bump the BOM to get the dependencies (where they recently added).

Unless needed I do not think we should bump the BOM.
If the plugin is part of the BOM then this is in essence done in the BOM project so doing it here makes it much harder for users to get a single fix in a plugin (as one fix could require upgrading every plugin that the BOM defines).  If the plugin is not in the BOM then maybe it should be?

Reply:
why? 
As far I understand this will not affect users directly but at least we will build the plugin with up2date versions of dependencies which could be interested to detect potential (future) issues?

Reply:
I get your point.
but haha by upgrading this I was acting as human dependabot :)
It's really interesting to see how we have created our own infinite loop and maybe a stackoverflow 🤣 by having the plugins itself declared in a bom they are using.... we even accelerate with dependabot and automatic release.